index.bs

```diff
@@ -1437,6 +1437,21 @@ to {{SensorErrorEventInit}}.
         1.  Let |result| be the result of invoking |type|'s [=threshold check algorithm=]
             with |reading| and |sensor|'s [=latest reading=].
         1.  If |result| is false, then abort these steps.
+    1.  If |reading|["timestamp"] [=map/exists=]:
+        <!-- The language below is similar to what is found in
+             https://w3c.github.io/hr-time/#timeorigin-attribute -->
+        1.  Set |reading|["timestamp"] to the result of converting its current value in an
+            [=implementation-defined=] way to an [=monotonic clock/unsafe current time=] using the
+            same [=monotonic clock=] that is shared by [=time origins=].
+
+            Note: The goal of this step is to ensure that a timestamp that may have been relative to
+            a different time origin is converted to a value that can be used in computations with
+            the same [=monotonic clock=] used by the operations described in [[HR-TIME]].
+    1.  Otherwise, [=map/set=] |reading|["timestamp"] to the [=unsafe shared current time=].
+
+        Note: In neither case is an [=monotonic clock/unsafe current time=] ever exposed to script.
+        {{Sensor/timestamp|Sensor.timestamp}} always returns a coarsened moment relative to a
```

Comment:
```suggestion
        {{Sensor/timestamp|Sensor.timestamp}} always returns a [=moment/coarsened moment=] relative to a
```

Reply:
Done, thanks. I've just kept it wrapped at 100 characters.